database/open_source/os-programs.json

```
@@ -0,0 +1,37 @@
+[
+    {
+      "name": "Google Summer of Code GSoC",
+      "description": "Google Summer of Code is a global, online mentoring program focused on introducing new contributors to open source software development.",
+      "url": "https://summerofcode.withgoogle.com/",
+      "category": "open-source",
+      "subcategory": "os-programs"
+    },
+    {
+      "name": "Open Source Initiative",
+      "description": "The Open Source Initiative is a foundation of the modern software ecosystem. We build a world where the freedoms and opportunities of Open Source software can be enjoyed by all.",
+      "url": "https://opensource.org/events/",
+      "category": "open-source",
+      "subcategory": "os-programs"
+    },
+    {
+      "name": "Free Software Foundation",
+      "description": "The Free Software Foundation (FSF) is a nonprofit with a worldwide mission to promote computer user freedom.",
+      "url": "https://www.fsf.org/events/aggregator/",
+      "category": "opensource",
+      "subcategory": "os-programs"
+    },
+    {
+      "name": "OWASP",
+      "description": "To be the global open community that powers secure software through education, tools, and collaboration.",
+      "url": "https://owasp.org/events/",
+      "category": "open-source",
+      "subcategory": "os-programs"
+    },
+    {
+      "name": "The Linux Foundation",
+      "description": "Empowering generations of open source innovators. Unlocking the value of shared technology Innovation, comes from everywhere.",
```

[suggestion] This description could be improved. Consider using the following questions to help you revise it:
1. What makes Linux Foundation great?
2. Why should open source enthusiasts join the program?
```suggestion
      "description": "Empowering generations of open source innovators. Unlocking the value of shared technology Innovation, comes from everywhere.",
```